pypesto/optimize/task.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import logging
+
+from ..engine import Task
+from ..objective import HistoryOptions
+from ..problem import Problem
+import pypesto.optimize
+
+logger = logging.getLogger(__name__)
+
+
+class OptimizerTask(Task):
```

Comment:
moved to pypesto.optimize in order to have self-contained submodules (and it did not work otherwise)